src/components/AllocatedChip/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react';
+import { Chip } from 'components/ui';
+
+type AllocatedChipProps = {
+  value: boolean | undefined;
+};
+
+export const AllocatedChipChip: FC<AllocatedChipProps> = (props) => {
+  const { value } = props;
```

Comment:
Could we change the name to isAllocated?

Reply:
I would also destructure it in the same line avoiding extra lines for each component. Looks cleaner IMO